util/design/gen-otp-img.py

```diff
@@ -57,6 +70,7 @@ def main():
         prog="gen-otp-img",
         description=wrapped_docstring(),
         formatter_class=argparse.RawDescriptionHelpFormatter)
+    parser.register('action', 'extend', ExtendAction)
```

Comment:
Do you really want to be able to do this?
```console
$ gen_otp_img.py --add-cfg foo --switch-x p --switch-y q --add-cfg bar
```

Reply:
The script has been constructed such that you can incrementally augment (and overwrite) existing configurations in the order the config files are specified (in your example, `bar` would be applied after `foo`). Hence yes, this is intended.

Reply:
Ooh I see. In that case you need the `extend` action. It is surprising that it is not well-established already.